Bio/SeqIO/AbiIO.py

```diff
@@ -353,6 +353,10 @@ def _get_string_tag(opt_bytes_value, default=None):
 
 def AbiIterator(handle, alphabet=None, trim=False):
     """Return an iterator for the Abi file format."""
+    # raise exception if file is empty
+    if not handle.readline():
+        raise ValueError("Empty file.")
+
```

Comment:
Can you move the raise to line below after the comment "handle empty file gracefully" please?